.github/workflows/core-build-test.yml

```
@@ -0,0 +1,106 @@
+        required: false
+        default: 'self-hosted'
+        type: string
+
+jobs:
+  build-and-test:
+    name: build and test
+    runs-on: ${{ inputs.runner }}
+    timeout-minutes: 120
+    steps:
+      - name: Create check run
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)